Source/WarpX.cpp

```diff
@@ -1512,20 +1510,19 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
 {
     // Declare nodal flags
     IntVect Ex_nodal_flag, Ey_nodal_flag, Ez_nodal_flag;
+    IntVect Bx_nodal_flag, By_nodal_flag, Bz_nodal_flag;
```

Comment:
is Bfield used even when LLG=TRUE?
Because we only use H and M
But I think we compute what B is somewhere

Reply:
yeah we compute B in `MacroscopicEvolveHM`